shared/components/splash-example.jsx

```diff
@@ -59,7 +59,7 @@ export default React.createClass({
                             # Photos:
                         </span>
 
-                        <select
+                        <select className="num-photos-select"
```

Comment:
Is this addition of this `className` just for testing purposes to have a hook?

If so, maybe it would be better to add `ref`s throughout the component hierarchy, that way we can access the React component, and if we want its we can use `getDOMNode()` on the component instance that the `ref` points to.